### ISSUE TITLE
Soldering the jumpers

### ISSUE THREAD

**Issue by pstapley** [raeedcho/chocofi-temper#24]
I am having trouble getting the solder to bridge the gap on the jumpers, any  suggestions?

**Comment by rmaced0** [comment]
What does it look like? Are you using Flux?

**Comment by pstapley** [comment]
Yeah I am using flux, I will get a picture for you. Actually I am not sure if my phone will focus that closely. The solder balls on the pads but won't cross the "v".

**Comment by rmaced0** [comment]
How much solder are you using? The readme images look like they use a lot. Try adding solder to both sides individually then join them

**Comment by pstapley** [comment]
A lot, the excess just sticks to the iron. I will give your suggestion a shot, thanks!

**Comment by rmaced0** [comment]
Something else you could try is use a jumper wire to join them, if you have any wire you can trim to size

**Comment by pstapley** [comment]
Yeah I was hoping not to resort to that but I will if I can't get it to work with just solder. Thanks!
